extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -72,7 +72,6 @@ public class DefaultAccessLogReceiver implements AccessLogReceiver, Runnable, Cl
 
     private long changeOverPoint;
     private String currentDateString;
-    private boolean forceLogRotation;
```

Comment:
could prevent this with 

- test coverage of 100% like in check.
- mutation testing, right?